website/screens/components/sidenav/code/SidenavCodePage.tsx

```diff
@@ -22,216 +22,211 @@ const sections = [
+              <th>Default</th>
+              <HeaderDescriptionCell>Description</HeaderDescriptionCell>
+            </tr>
+            <tr>
+              <td>title: string</td>
+              <td></td>
+              <td>The title of the sidenav group.</td>
+            </tr>
+            <tr>
+              <td>collapsable: boolean</td>
+              <td>
+                <Code>false</Code>
+              </td>
+              <td>
+                If true the sidenav group title will be considered a button and
+                the group will be collapsable.
+              </td>
+            </tr>
+            <tr>
+              <td>icon: SVG | string</td>
+              <td></td>
+              <td>The icon to be displayed next to the title of the group.</td>
+            </tr>
+            <tr>
+              <td>children: node</td>
+              <td></td>
+              <td>
+                The area inside the sidenav group. This area can be used to
+                render sidenav links.
+              </td>
+            </tr>
+          </DxcTable>
+        ),
+      },
+    ],
+  },
+  {
+    title: "DxcSidenav.Link",
+    content: (
+      <DxcParagraph>
+        As with the <Code>DxcLink</Code> component, we decided to make our
+        Sidenav link component a styled HTML anchor that can be used with any
+        React-based router. You can check the{" "}
+        <Link href={"/components/link/"} passHref>
+          <DxcLink>Link</DxcLink>
+        </Link>{" "}
+        for more information regarding this.
+      </DxcParagraph>
+    ),
+    subSections: [
       {
-        title: "DxcSidenav.Section",
+        title: "Props",
         content: (
-          <DxcParagraph>
-            This must be defined as children of the Sidenav. It helps to
-            separate groups and links into different agroupations. It has the
-            following props:
-          </DxcParagraph>
+          <DxcTable>
+            <tr>
+              <th>Name</th>
+              <th>Default</th>
+              <HeaderDescriptionCell>Description</HeaderDescriptionCell>
+            </tr>
+            <tr>
+              <td>href: string</td>
+              <td></td>
+              <td>Page to be opened when the user clicks on the link.</td>
+            </tr>
+            <tr>
+              <td>newWindow: boolean</td>
+              <td>
+                <Code>false</Code>
+              </td>
+              <td>If true, the page is opened in a new browser tab.</td>
+            </tr>
+            <tr>
+              <td>icon: SVG | string</td>
+              <td></td>
+              <td>
+                Element or path used as the icon that will be placed to the left
+                of the link text.
+              </td>
+            </tr>
+            <tr>
+              <td>selected: boolean</td>
+              <td>
+                <Code>false</Code>
+              </td>
+              <td>If true, the link will be marked as selected.</td>
```

Comment:
Maybe we can add a recommendation regarding this prop and how the group if collapsable should have this prop accessible in a direct child.

Reply:
Is hard to make the prop accessible if they are using a custom or external link (forcing them to add the `selected` prop just for our implementation to use it and not theirs is a bit strange and difficult to understand). But I agree that we should notify them that if they don't use our Sidenav link this can lead to some functionality loss.